sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -1498,9 +1538,7 @@ static ReadChangeStream create() {
 
     abstract BigtableConfig getBigtableConfig();
 
-    abstract @Nullable Timestamp getStartTime();
-
-    abstract @Nullable Timestamp getEndTime();
```

Comment:
we don't care about endTime anymore?

Reply:
Yes. We don't support it for now. Removed in https://github.com/apache/beam/pull/25474